services/ffc/ffc-demo/ffc-demo-claim-service/pre-deploy-kustomize.yaml

```diff
@@ -30,6 +30,9 @@ spec:
       POSTGRES_DB: "ffc-demo-claim"
       POSTGRES_PORT: "5432"
       POSTGRES_SCHEMA_NAME: "public"
+      #These variables will need to be populated after onboarding automation when flux manifests are generated
```

Comment:
Great PR. My question is that how much of what is added to the Flux Manifests needs to be added or is missing from the scaffolder for Flux Manifests? @kbabigumira @aadev1 @jeevankuduvaravindran

Reply:
@DaRoszko For now, the below variables needs to come from scaffolder which are missing now.
 postgres AD groups, postgres DB name and schema name.